### PR TITLE
feat: [DSSC#8669jr6x4] Integration de la page Landing Page

### PR DESCRIPTION
**Pull request by aertyuiop12345**
Closes #

#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [ ] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [ ] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    @Inject
+    MainMenuViewModel.Factory viewModelFactory;
+
+    @Inject
+    CurrentProjectViewModel.Factory currentProjectViewModelFactory;
+
+    @Inject
+    SettingsProvider settingsProvider;
+
+    private MainMenuViewModel mainMenuViewModel;
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java]: Innutilisé. A supprimer

File: collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+public class LandingPageActivity extends CollectAbstractActivity{
+    //button
+    private Button sondageButton;
+    private Button messageButton;
+    private Button articleButton;
+    private Button centreHospitalButton;
+    private Button cycleButton;
+
+    @Inject
+    MainMenuViewModel.Factory viewModelFactory;
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java]: Inutilisé. A supprimer

File: collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        ProjectIconView projectIconView = (ProjectIconView) projectsMenuItem.getActionView();
+        projectIconView.setOnClickListener(v -> onOptionsItemSelected(projectsMenuItem));
+        projectIconView.setContentDescription(getString(R.string.projects));
+
+        return super.onPrepareOptionsMenu(menu);
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.main_menu, menu);
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java]: Le menu utilisé sur cette page n'est pas celui-ci. A corriger !

File: collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+    @Inject
+    CurrentProjectViewModel.Factory currentProjectViewModelFactory;
+
+    @Inject
+    SettingsProvider settingsProvider;
+
+    private MainMenuViewModel mainMenuViewModel;
+
+    private CurrentProjectViewModel currentProjectViewModel;
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java]: A supprimer

**Review by Baom92 — CHANGES_REQUESTED** [review]

**Comment by Baom92** [comment]
Il y a toujours des variables inutilisées dans le code. Android Studio les marques. Juste suivre les recommendations d'Android studio

File: collect_app/src/main/res/layout/landing_page.xml [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:layout_margin="@dimen/margin_standard">
+
+<!--            <androidx.fragment.app.FragmentContainerView-->
```
Comment (Baom92) [inline collect_app/src/main/res/layout/landing_page.xml]: Supprimer les codes comentés

**Review by Baom92 — CHANGES_REQUESTED** [review]

**Review by Baom92 — APPROVED** [review]

**Review by jgnacadja — APPROVED** [review]

**Pull request merged**